api/hello_test.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestHelloHandler(t *testing.T) {
+
+	Convey("Given a Hello handler ", t, func() {
+		helloHandler := HelloHandler()
```

Comment:
This is failing (its a bug in the auto-build script) - add context:

Reply:
I'm new to golang. Thanks for pointing out that the test was failing. However, although I've added the context, I'm not sure that it's passing yet? Perhaps I've missed something else..

Reply:
No problem - yes, its passing now -> https://concourse.onsdigital.co.uk/builds/141152

and as shown below:

![image](https://user-images.githubusercontent.com/77387151/109977127-68613e80-7cf4-11eb-8033-68c0f0d3e570.png)